src/pages/Resources/Resources.scss

```diff
@@ -0,0 +1,60 @@
+.resources {
+  background-color: #ebebf5;
+
+  &__container {
+    display: none;
+  }
+
+  .desktop__container {
+    display: flex;
+    height: 100vh;
+
+    &--right {
+      width: 100%;
+      height: auto;
+    }
+  }
+
+  .data-wrap {
+    overflow-y: auto;
```

Comment:
This has a scroll bar to the left aswell as up and down we only want a scroll bar for up and down can you take a look